testbed/integration/config_examples_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	"go.opentelemetry.io/collector/pdata/ptrace"
+	semconv "go.opentelemetry.io/collector/semconv/v1.18.0"
```

Comment:
This file is already getting rather long with a bunch of imports that only apply to certain cases, would it make sense to split the tests into individual files/modules that map 1:1 top the config_example yamls?

Reply:
I'm not too concerned about the dependencies here since this is just a test. I don't care either way exactly which files we put the test cases into, but I've stuck with keeping them in the same file for now since we only have a single test case per config example.